rqt_topic/src/rqt_topic/topic_widget.py

```diff
@@ -347,9 +347,21 @@ def set_selected_topics(self, selected_topics):
                                                          len(selected_topics)))
         self._selected_topics = selected_topics
 
+    # TODO(Enhancement) Save/Restore tree expansion state
+    def save_settings(self, plugin_settings, instance_settings):
+        header_state = self.topics_tree_widget.header().saveState()
+        instance_settings.set_value('tree_widget_header_state', header_state)
+        instance_settings.set_value('settings_exist', True)
```

Comment:
Why is this boolean flag necessary? The existence / non-existence of `tree_widget_header_state` should be enough, no?